src/layouts/Media/Media.tsx

```diff
@@ -288,27 +367,32 @@ export const Media = (): JSX.Element => {
                             color="interaction.sub.default"
                             ml="0.5rem"
                           >
-                            {selectedImages.length}
+                            {selectedMedia.length}
                           </Badge>
                         </Menu.Button>
                         <Menu.List>
+                          {/* FIXME: To add back when flow is available
```

Comment:
sanity check here, is this blocking push to production?

Reply:
Nope, they just won't see those options but the existing flow still exists where they can move their files individually.